druid/examples/hello.rs

```diff
@@ -12,62 +12,54 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+//! This is a very small example of how to setup a druid application.
+//! It does the almost bare minimum while still being usefull.
```

Comment:
```suggestion
//! This is a very small example of how to setup a druid application.
```

I think short concise comments are good!